doc/manpages/man1/ad.1.xml

```diff
@@ -7,7 +7,9 @@
 
     <refmiscinfo class="date">05 Dec 2015</refmiscinfo>
 
-    <refmiscinfo class="source">@NETATALK_VERSION@</refmiscinfo>
+    <refmiscinfo class="source">Netatalk</refmiscinfo>
+    <refmiscinfo class="manual">Netatalk AFP fileserver Manual</refmiscinfo>
```

Comment:
This is just my OCD talking, but is there any particular reason only the word "fileserver" isn't capitalized, and written as one word? A quick grep in the codebase suggests that "file server" is more common in the documentation.

Reply:
Done